sormas-api/src/main/java/de/symeda/sormas/api/utils/AgeGroupUtils.java

```diff
@@ -75,38 +83,46 @@ private static String getAgeGroupTimeUnitCaption(String ageGroupTimeUnit) {
 	}
 
 	public static List<String> convertToList(String ageGroupsString) {
+
 		if (StringUtils.isBlank(ageGroupsString)) {
 			return null;
 		}
 
-		try {
-			final List<String> ageGroupList = Stream.of(ageGroupsString.split(",")).collect(Collectors.toList());
-			ageGroupList.forEach(s -> validateAgeGroup(s));
-			return ageGroupList;
-		} catch (IllegalArgumentException e) {
-			throw new RuntimeException(
-				"Content of ageGroupsString is not a valid list of ageGroups, or one of the ageGroups in the String does not match a valid ageGroup: "
-					+ ageGroupsString);
-		}
+		final List<String> ageGroupList = Stream.of(ageGroupsString.split(",")).collect(Collectors.toList());
+		final List<String> invalidAgeGroupsList = new ArrayList<>();
+		ageGroupList.forEach(s -> {
```

Comment:
I'd suggest to user removeIf: https://www.geeksforgeeks.org/arraylist-removeif-method-in-java/